htmlcss/task2/index.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Document</title>
+    <link rel="stylesheet" href="header.css">
+    <link rel="stylesheet" href="info-container.css">
+    <link rel="stylesheet" href="adv-container.css">
+    <link rel="stylesheet" href="art-container.css">
+    <link rel="stylesheet" href="footer.css">
+    <link rel="stylesheet" href="fonts.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="sub-wrapper">
+            <header>
+                <div class="top-header">
+                    <img src="images/logo.png" alt="logo">
+                    <p class="top-header__logo">sitename</p>
+                    <p class="top-header__text">lorem ipsum dolor sit amet vivamus</p>
+                </div>
+                <div class="nav-container">
+                    <nav>
+                        <div class="nav__list">
+                            <a href="">home</a>
+                            <a href="">about</a>
+                            <a href="">services</a>
+                            <a href="">solutions</a>
+                            <a href="">portfolio</a>
+                            <a href="">contact</a>
+                        </div>
+                    </nav>
+                    <div class="bottom-header">
```

Comment:
Please move this to `<section class="banner">...</section>`
Also, take a look on other divs and probably wrap/replace them with sections